identity_iota_core/packages/iota_identity/sources/identity.move

```diff
@@ -23,6 +24,10 @@ module iota_identity::identity {
     const EInvalidThreshold: u64 = 2;
     /// The controller list must contain at least 1 element.
     const EInvalidControllersList: u64 = 3;
+    /// Idenity cannot does not need an upgrade.
+    const ENoUpgrade: u64 = 4;
```

Comment:
Hm, would something like `EAlreadyUpgraded` make more sense here? For some reason, I find "no upgrade" a bit confusing. But this could also be due to me being confused, or that the name might make more sense for me with future updates. ^^

Also, the comment has some artifacts from updating the wording. ^^

Reply:
The thing is that if you can't upgrade it means that a next version of the package doesn't exist rather than "this identity has already upgraded". I fixed the comment wording but kept the name.

Reply:
Ah, makes sense. Understood the check `assert!(self.version < PACKAGE_VERSION, ENoUpgrade);` to prevent the update if the identity is already at version `PACKAGE_VERSION`.